api/src/services/workspace-old/index.ts

```diff
@@ -0,0 +1,250 @@
+// /* eslint-disable @typescript-eslint/explicit-function-return-type */
```

Comment:
Still needed ?

Reply:
Yup, I remove it in the next PR when the `record` service will be implemented.

Reply:
ok